lighthouse-core/test/fixtures/fraggle-rock/reports/sample-flow-result.json

```diff
@@ -3402,19 +3402,6 @@
               ]
             }
           },
-          "external-anchors-use-rel-noopener": {
```

Comment:
Good reminder for me to rebaseline this before release. It might cause issues for you to make changes in this PR though.